app/admin/api/v3/context.rb

```diff
@@ -16,18 +18,35 @@ def scoped_collection
     end
   end
 
+  includes [
+    :country,
+    :commodity,
+    :resize_by_attributes,
+    :recolor_by_attributes
+  ]
```

Comment:
we can add `top_profiles` here as well

Reply:
and I guess, as we're using `includes` now directly, we no longer need `scoped_collection`

Reply:
excellent point, thank you!